### PR TITLE
TextArea Components and Static Text Pages

### PR DESCRIPTION
**Pull request by ItsAnkan**
This PR has the components for static text pages, and the converted pages using those components

Covers `/terms`, `/privacy`, `/about` and `/events/[event-name]`

### CODE REVIEW

**Review by Goose-Of-War — COMMENTED** [review]
Shameless copying of the base components, eh?

**Comment by ItsAnkan** [comment]
I edited much stuff ;-;

**Comment by Goose-Of-War** [comment]
image
_smile in pain: Ankan edition_

**Comment by Goose-Of-War** [comment]
That should explain why there no images in OACQ

**Comment by ItsAnkan** [comment]
Fixed

**Comment by ItsAnkan** [comment]
Fixed

**Comment by Goose-Of-War** [comment]
Event pages are missing headings -_-

**Comment by Goose-Of-War** [comment]
Also, why you clone HeadContent component if you ain't using it everywhere...

**Comment by Goose-Of-War** [comment]
SMH

**Comment by Goose-Of-War** [comment]
You forgot Cosplay 23 ;-;

File: pages/events/cosplay23.tsx
Comment (Goose-Of-War) [inline pages/events/cosplay23.tsx:1]: Heading where?

**Review by Goose-Of-War — CHANGES_REQUESTED** [review]
fix onegai

**Review by Goose-Of-War — APPROVED** [review]
I blame Ankan if anything breaks

**Pull request merged**